libp2p/CHANGELOG.md

```diff
@@ -14,10 +14,15 @@
 - Introduce `libp2p::perf` module.
   See [PR 3693].
 
+- Deprecate the `mplex` feature.
+  The recommended baseline stream multiplexer is `yamux`.
+  See [PR 3689].
+
 [PR 3386]: https://github.com/libp2p/rust-libp2p/pull/3386
 [PR 3580]: https://github.com/libp2p/rust-libp2p/pull/3580
 [PR 3590]: https://github.com/libp2p/rust-libp2p/pull/3590
 [PR 3693]: https://github.com/libp2p/rust-libp2p/pull/3693
+[PR 3689]: https://github.com/libp2p/rust-libp2p/pull/3689
```

Comment:
This needs to be moved a new version, plus bump the minor version of `libp2p` please. See https://github.com/libp2p/rust-libp2p/blob/master/docs/release.md#make-changelog-entries.

Reply:
@thomaseizinger Done, I did a minor bump in the `CHANGELOG.mod` of `libp2p` folder as `0.52.3` and put the related information here.